bower.json

```diff
@@ -20,6 +20,6 @@
     "tests"
   ],
   "dependencies": {
-    "file-saver": "^1.3.3"
+    "file-saver": "=1.3.4"
```

Comment:
`=` is not necessary

Reply:
I know it's equivalent to `"1.3.4"`, but I intentionally chose to put it there to specify what it's doing. Because I'm not sure if some of our newer compatriots would necessarily know the lock at 1.3.4 was intentional.

Reply:
But don't worry. I've logged a tech debt ticket. :'(